packages/alfa-css/src/value/numeric/dimension.ts

```diff
@@ -1,20 +1,18 @@
 import { Comparable, Comparison } from "@siteimprove/alfa-comparable";
+import { Hash } from "@siteimprove/alfa-hash";
 
-import {
-  Dimension as BaseDimension,
-  Numeric as BaseNumeric,
-} from "../../calculation/numeric";
-import { Convertible } from "../../unit";
+import { Numeric as BaseNumeric } from "../../calculation/numeric";
+import { Convertible, Unit } from "../../unit";
 import { Value } from "../value";
 
 import { Numeric } from "./numeric";
-import { Hash } from "@siteimprove/alfa-hash";
 
 /**
  * @public
  */
-export type Dimension<T extends BaseNumeric.Dimension = BaseNumeric.Dimension> =
-  Dimension.Calculated<T> | Dimension.Fixed<T>;
+export type Dimension<T extends Type = Type> =
```

Comment:
Well, it is the value that is going to go in the `_type` variable that is used to discriminate the union 😅 
I think it's also OK because it is only a local one, not an exported one.